setup.sh

```diff
@@ -33,3 +39,6 @@ git clone git@github.com:${FORK}/TreeMaker.git -b ${BRANCH}
 scram b -j 8
 cd TreeMaker/Production/test/condorSub/
 python $CMSSW_BASE/src/Condor/Production/python/linkScripts.py
+ln -s ${CMSSW_BASE}/src/NNKit/data/ak8/full/preprocessing.json ${CMSSW_BASE}/src/TreeMaker/Production/test/.
```

Comment:
it would be better to link these into `TreeMaker/Production/test/data` (where we keep other various input files)